adoc/admin-kubernetes-changes.adoc

```diff
@@ -0,0 +1,74 @@
+== From Kubernetes 1.17 to 1.18
+
+This documentation page lists all the behavioral changes and API deprecations that will happen from Kubernetes 1.17 to 1.18. This is going to be relevant when migrating to {productname} 4.3.x, since it will bring Kubernetes 1.18.
+
+=== Architectural changes in {productname}
+
+`hyperkube` has been removed from the Kubernetes project and is now community maintained. That being said, its future is uncertain, and it was no longer really needed for {productname}. Thus, we have decided to remove this binary starting from {productname} 4.3.0 and we are now providing binaries and container images for each Kubernetes component.
+
+This should not affect you directly: it will be transparent. That being said, if you relied on `hyperkube` specifically for some reason, take this into account.
+
+=== API deprecations
+
+The following APIs have been deprecated:
+
+- All resources under *apps/v1beta1* and *apps/v1beta2*: from now on use *apps/v1* instead.
```

Comment:
nit: that sound very harsh, "from now on use" -> "please use" or "you must now use" ?